cypress/integration/locators.spec.ts

```diff
@@ -0,0 +1,29 @@
+import {
+    PersonalInformation   
+  } from '../page';
+  
+  describe('Fill Form', () => 
+  {
+    const personalInformation: PersonalInformation = new PersonalInformation();
+    describe('given a user enters the qavbox page', () => 
+    {
+      before(() => {
+        cy.visit('https://qavbox.github.io/demo/signup/');
+    })
+    
+    describe('when the user fills and submits the form', () => {
+      before(() => 
+    {
+        personalInformation.fillForm();
+    }) 
+
+    describe('then the information is submited.', () => 
+    {
+      it('then the information is submited.',() => 
+      {
+        personalInformation.test();
+      }) 
+    })   
+  }) 
```

Comment:
```suggestion
    it('then the information should be submited.',() =>
    {
      personalInformation.test();
    })
  })
```
anything else that tells us that the test is successful?